pdf_to_csv.py

```diff
@@ -70,16 +74,32 @@ def generate_csv_file(self, input_pdf_filepath, out_csv_filepath,
                                                             page_num,
                                                             is_header,
                                                             identify_columns,
-                                                            check_page_rotation)
+                                                            check_page_rotation,
+                                                            measure_file_obj)
             if page_table_data:
                 out_file_obj.write("\n%s" % page_table_data)
             out_file_obj.write("\n%s" % self.page_break)
         out_file_obj.close()
         self.process_csv_file(out_csv_filepath)
 
+    def init_measure_log_file(self, measure_file_name):
+        '''Initialize a file object and add headers.
+
+        Args:
+            - measure_file_name (string): name/path of the file in string
+                format.
+
+        Returns:
+            The file object.
+        '''
+        measure_file_obj = open(measure_file_name, 'w')
```

Comment:
Why pass another file, can be included in the default logs with just a separate parameter which easy to grep? "MONITORING LOG" or "PERF LOG"